.github/workflows-disabled/coverage.yml

```diff
@@ -32,22 +32,33 @@ jobs:
           submodules: recursive
 
       - uses: ./.github/actions/deps
+      - uses: ./.github/actions/hugepages
```

Comment:
Note that this file workflow is disabled and will be adjusted in an upcoming PR.